src/test/java/uk/gov/companieshouse/api/strikeoffobjections/service/impl/EmailServiceTest.java

```diff
@@ -96,6 +88,96 @@ void sendObjectionSubmittedCustomerEmail() throws ServiceException {
         assertTrue(data.containsValue("Company: " + COMPANY_NUMBER));
         assertTrue(data.containsValue(OBJECTION_ID));
         assertTrue(data.containsValue(attachments));
+    }
+
+    @Test
+    void sendObjectionSubmittedDissolutionEmailsWalesJurisdiction() throws ServiceException {
+        when(dateTimeSupplier.get()).thenReturn(LOCAL_DATE_TIME);
+        when(config.getEmailRecipientsCardiff()).thenReturn(EMAIL_RECIPIENTS_CARDIFF_TEST);
+        Objection objection = Utils.getTestObjection(OBJECTION_ID);
+        Utils.getTestAttachments().forEach(objection::addAttachment);
+
+        emailService.sendObjectionSubmittedDissolutionTeamEmail(
+                REQUEST_ID,
+                Utils.getDummyCompanyProfile(COMPANY_NUMBER, "wales"),
+                objection
+        );
+
+        ArgumentCaptor<EmailContent> emailContentArgumentCaptor = ArgumentCaptor.forClass(EmailContent.class);
+        verify(kafkaEmailClient, times(3)).sendEmailToKafka(emailContentArgumentCaptor.capture());
```

Comment:
The captured arguments aren't being asserted against so it doesn't test if the method is behaving correctly. For this test we need to check that england/wales emails are sent. To do that we'd need to check the contents of the arguments captured to see if they contain the correct values (email addresses etc..)

For multiple calls, this is an example from the Mockito javadoc:
`ArgumentCaptor<Person> peopleCaptor = ArgumentCaptor.forClass(Person.class);
verify(mock, times(2)).doSomething(peopleCaptor.capture());

List<Person> capturedPeople = peopleCaptor.getAllValues();
assertEquals("John", capturedPeople.get(0).getName());
assertEquals("Jane", capturedPeople.get(1).getName());`

more examples and docs - https://javadoc.io/static/org.mockito/mockito-core/3.4.6/org/mockito/ArgumentCaptor.html

This applies to the other tests as well.

Reply:
Updated testing.